### PR TITLE
ENH: Adding fields parameter to specify list of writing to file.

### PR DESCRIPTION
**Pull request by zssherman**

### CODE REVIEW

**Comment by zssherman** [comment]
going to change name, fields = none almost seems like no fields included by default going to do a change.

File: pyart/io/cfradial.py [hunk 1]
```diff
@@ -437,6 +437,8 @@ def write_cfradial(filename, radar, format='NETCDF4', time_reference=None,
         NetCDF format, one of 'NETCDF4', 'NETCDF4_CLASSIC',
         'NETCDF3_CLASSIC' or 'NETCDF3_64BIT'. See netCDF4 documentation for
         details.
+    fields : list, optional
+        Included fields for the generated NETCDF4 file. Default is None.
```
Comment (mgrover1) [inline pyart/io/cfradial.py:441]: Can you rephrase as "Fields to write out to NETCDF file"

**Review by mgrover1 — COMMENTED** [review]
One minor comment/suggestion

**Comment by zssherman** [comment]
@mgrover1 did your changes but also changed fields parameter to include_fields to be consistent with other pyart io functions

**Review by mgrover1 — APPROVED** [review]
